### PR TITLE
Spacemens will loudly announce that they are changing magazines

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**

### CODE REVIEW

**Comment by Zilenan91** [comment]
JAMMING IN A CLIP WATCH MY BACK

**Comment by GunHog** [comment]
Seriously?? This looks like something @Iamgoofball would make.

**Comment by KorPhaeron** [comment]
OOC told me to do it

**Comment by PKPenguin321** [comment]
10/10

**Comment by as334** [comment]
:+1:

**Comment by newfren** [comment]
:+1:

**Comment by MMMiracles** [comment]
please yes

**Comment by Iamgoofball** [comment]
@GunHog You are right, this is exactly something I would do.

:+1:

**Comment by Steelpoint** [comment]
> last surviving Op hiding in a locker with a empty gun

> reloads his gun

> "RELOADING HERE"

Totallywontgetpeoplekilled/10

**Comment by optimumtact** [comment]
that does sound like it would make for an awesome story though.

**Comment by MrPerson** [comment]
Surprisingly useful even if it's goofball-y

**Comment by phil235** [comment]
The message is being said after you put a new magazine in, so "OUT OF AMMO!!!" is counterintuitive.

**Comment by CosmicScientist** [comment]
Phil makes a good point. Unless you could put in the out of ammo at the point when the gun *first* goes click/bleeps the warning.

Also this doesn't affect tacticool reloads if you want to evade the reloading whilst hiding but shouting words shenanigans.

After this all I'd want is a spaceman to say "Cash here, grab it all" and "Loadsa money" when throwing the space cash.

**Comment by Alek2ander** [comment]
"FIRE IN THE HOLE"

**Comment by KorPhaeron** [comment]
@Alek2ander 

Grenades are next don't worry

**Comment by Bawhoppen** [comment]
as meme as this is, its actually really dumb :-1:

**Comment by Steelpoint** [comment]
So long as this does not occur during a tactical reload then I think its fine, my main concerns was not being able to stealth load yet this seems to only occur if you manually reload.

However it may seem very dumb if your just removing a magazine outside of combat only to yell out for  no reason.

Prehaps if this system can be tied only into people with certain roles, like Nuke Ops, then it'll make more sense. 

Ehh, I just think this system might result in people screeching their reloading in situations where it makes no sense to.

**Comment by KorPhaeron** [comment]
Making it tactical reload only would make more sense honestly.

**Comment by KorPhaeron** [comment]
Whole PR is a joke anyway so if maintainers think it will seriously impact gameplay or something I'll just close it

**Comment by RemieRichards** [comment]
:-1: :x: :100:

**Comment by Alek2ander** [comment]
The gun should send a message into the flops radio channel, like some sort of tiny beepsky. This is the future, damnit, where are my smart-assault-shotguns?

**Comment by Cebasiz** [comment]
shadowrun-esque selectable gun personalities when

i want my cutesy anime light machine gun NOW DAMMIT

**Comment by Shadowlight213** [comment]
:+1: improves RP via communication during firefights

**Comment by The-Albinobigfoot** [comment]
I feel as though this is one big joke everyone but me is in on.
:-1:  :100:

**Comment by Iamgoofball** [comment]
:+:1

**Comment by Bawhoppen** [comment]
ye its a meme
pls don't actually merge holy cuck

**Comment by optimumtact** [comment]
merge it :+1:

File: code/modules/projectiles/guns/projectile/automatic.dm [hunk 1]
```diff
@@ -48,6 +48,7 @@
 			chamber_round()
 			A.update_icon()
 			update_icon()
+			user.say(pick("I'M RELOADING!!!", "NEW MAG IN!!!", "CHANGING MAG!!!", "COVER ME I'M RELOADING!!!", "OUT OF AMMO!!!", "RELOADING!!!"))
```
Comment (Niknakflak) [inline code/modules/projectiles/guns/projectile/automatic.dm]: There should be a comma in between COVER ME and I'M RELOADING
COVER ME, I'M RELOADING

**Comment by Niknakflak** [comment]
Hilarious but dumb

**Comment by Aloraydrel** [comment]
JUST DO IT

**Comment by GunHog** [comment]
You know that Raz, TK, or Dunc will merge this out of PURE LULZ if this hits the 24hr mark, right?

**Comment by PKPenguin321** [comment]
>Should have them shout when applying bruise/ointment

GOTTA PATCH UP, COVER ME!!

**Comment by optimumtact** [comment]
have them shout when they put on shoes

**TYING MY SHOE HERE, COVER ME!!**

**Comment by PKPenguin321** [comment]
**SWITCHING HANDS, COVER ME!!**

**Comment by optimumtact** [comment]
**EATING THIS SNACK HERE, COVER ME!!**

**Comment by Bawhoppen** [comment]
add fuel to janicarts and secways just so they can yell **BINGO FUEL!!**

**Comment by The-Albinobigfoot** [comment]
CHANGING INTENTS, STEER CLEAR!!

**Comment by Bawhoppen** [comment]
did not bold 0/10 smh

**Comment by The-Albinobigfoot** [comment]
what are you talking about dude, are you blind, like lmao

**Comment by Bawhoppen** [comment]
edits are cheating i have it cached

**Comment by The-Albinobigfoot** [comment]
I can't upload my screencap for some reason, but bro, trust.

**Comment by tkdrg** [comment]
> You know that Raz, TK, or Dunc will merge this out of PURE LULZ if this hits the 24hr mark, right?

That's because this is actually good.

**Comment by Bawhoppen** [comment]
>actually good 

>literally most meme PR of all time

**Comment by The-Albinobigfoot** [comment]
>PK bolds his first example surreptitiously.

Not so fast.

**Comment by The-Albinobigfoot** [comment]
First person perspective and corner leaning when?

**Comment by GunHog** [comment]
<b>AIRLOCK OPENED: CAPTAIN'S OFFICE</b>

**Comment by PKPenguin321** [comment]
actually when i posted mine it was unbolded for like 3 seconds but i ninja'd it to bold, so it basically was bold the whole time
fight me btw

**Comment by optimumtact** [comment]
**OPENING BACKPACK, GO ON WITHOUT ME**

**Comment by Shadowlight213** [comment]
we need callouts for more stuff. Like "**BREACHING! GET CLEAR!**" for c4

**Comment by RemieRichards** [comment]
literally nothing is preventing you from saying these phrases yourself.
This PR is stupid.

**Comment by Bawhoppen** [comment]
its a meme 

rolling for close

**Comment by KorPhaeron** [comment]
I can't type this stuff mid combat

**Comment by RemieRichards** [comment]
@KorPhaeron then git gud or put up with it.

@Bawhoppen 

4+ for close:

d6

**Comment by KorPhaeron** [comment]
Memes win

**Comment by Bawhoppen** [comment]
as  a wise man once said, merging is technically closing too

**Comment by Cheridan** [comment]
we should just make ss13 turn-based so that people have time to type out their own custom combat messages

**Comment by KorPhaeron** [comment]
@Cheridan 

So switch back to FEA?

File: code/modules/projectiles/guns/projectile/automatic.dm (hunk 1 above)
Comment (duncathan) [inline code/modules/projectiles/guns/projectile/automatic.dm]: >tfw when someone "corrects" grammar by telling someone to insert a comma splice

It should have a semicolon there; they're two independent clauses

Comment (duncathan) [inline code/modules/projectiles/guns/projectile/automatic.dm]: Also remove the out of ammo one

Comment (Bawhoppen) [inline code/modules/projectiles/guns/projectile/automatic.dm]: duncathan you can't actually be critiquing this meme

**Comment by KorPhaeron** [comment]
@duncathan 

Should I make this tactical reloads only?

**Comment by Bawhoppen** [comment]
just close it please its so fucking dumb honestly, the meme can only go on so long

**Comment by vista-** [comment]
It's 2552, for god's sake. At least make the gun itself say/beep it.

**Comment by duncathan** [comment]
@KorPhaeron yeah tactical reloads only is a good idea
That way stealth can still happen

**Comment by Aloraydrel** [comment]
prone stance/ kill streaks when?

**Comment by Bawhoppen** [comment]
let us reserve this PR for getting our autistic cravings out, and then go on and function like normal

**Comment by Chocobro** [comment]
:)

**Comment by tkdrg** [comment]
HEY IT WASN'T ME THIS TIME

**Comment by optimumtact** [comment]
**MERGING THIS PR HERE, COVER ME**

**Comment by GunHog** [comment]
> You know that Raz, TK, or Dunc will merge this out of PURE LULZ if this hits the 24hr mark, right?

I WARNED YOU!!!

**Comment by tkdrg** [comment]
it wasn't 24h

**Comment by PKPenguin321** [comment]
**FEEDING OFF THE HATER'S SALT, COVER ME!!**

**Comment by Incoming5643** [comment]
#14018

**Comment by duncathan** [comment]
@GunHog
every decision i make is cool and calculated.
cool and calculated, that is dunc.
cool.

**Comment by newfren** [comment]
calculated

**Comment by Bawhoppen** [comment]
can we revert this yet

**Comment by optimumtact** [comment]
**COVER ME, I'm REVERTING**

**Comment by Zilenan91** [comment]
WATCH MY BACK, DANK MEMES INCOMING

**Comment by duncathan** [comment]
@zilenan91 more like dunc memes amirite

**Comment by Zilenan91** [comment]
ayy

**Comment by PKPenguin321** [comment]
What a good pr

**Comment by Bawhoppen** [comment]
wrong

**Comment by duncathan** [comment]
cool. calculated.

**Pull request merged**